### PR TITLE
Kevinloritsch/About

### PR DESCRIPTION
**Pull request by Kevinloritsch**
image

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = {
           "white-100": "#FFFFFF",
           "white-200": "#F1F1F1",
           "gray-100": "#888787",
+          black: "#3C3E64",
```
Comment (fardinzam) [inline tailwind.config.js:20]: add quotes around black

File: src/components/landing/About.jsx [hunk 2]
```diff
@@ -1,5 +1,19 @@
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="bg-gradient-to-r from-senior-blue-200 to-senior-blue-100">
```
Comment (fardinzam) [inline src/components/landing/About.jsx:3]: the blue-200 is a little too dark, just add a new blue color that's a little lighter

**Comment by Kevinloritsch** [comment]
image

File: tailwind.config.js (hunk 1 above)
Comment (Kevinloritsch) [reply tailwind.config.js:20]: When I run `npm run format` it removes the quotes, but it won't let be commit without running `npm run format`.

Comment (fardinzam) [reply tailwind.config.js:20]: oh that's weird, i'll take a look at it later but for now it's good

File: src/components/landing/About.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const About = () => {
   return (
-    <div className="bg-gradient-to-r from-senior-blue-200 to-senior-blue-100">
+    <div className="bg-gradient-to-r from-[#B7E3F0] to-senior-blue-100">
```
Comment (fardinzam) [inline src/components/landing/About.jsx:3]: could you add this color in the theme

Comment (Kevinloritsch) [reply src/components/landing/About.jsx]: Just added it!

**Pull request merged**